crates/isograph_schema/src/validate_client_field.rs

```diff
@@ -385,6 +399,7 @@ fn validate_field_type_exists_and_is_scalar(
                     )),
                 }
             }
+            FieldType::ClientField(ClientType::ClientPointer(_)) => todo!(),
             FieldType::ClientField(ClientType::ClientField(client_field_id)) => {
                 validate_client_field(
```

Comment:
I wonder if we can make client pointers and fields both implement the same interface, and have `validate_client_field` accept an `&impl ThatCommonInterface`, since presumably the validation is identical

But we could also duplicate it, if it's simple

Anyway, let's have `todo!("validating client pointers is not yet implemented")`